assets/src/edit-story/components/library/panes/shared.js

```diff
@@ -36,76 +30,60 @@ const Tabs = styled.ul`
   list-style: none;
 `;
 
-const Tab = styled.li`
-  width: 64px;
-  height: 100%;
-  color: ${({ theme }) => theme.colors.fg.v1};
+const Icon = styled.div`
+  width: 60%;
+  height: 80%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
 `;
 
-const Icon = styled.a`
-  color: inherit;
+const Tab = styled.li.attrs(({ isActive }) => ({
+  tabIndex: isActive ? 0 : -1,
+  role: 'tab',
+}))`
+  width: 72px;
+  height: 100%;
+  color: ${({ theme }) => theme.colors.fg.v1};
   background: ${({ isActive, theme }) =>
     isActive ? theme.colors.bg.v4 : 'transparent'};
-  height: 100%;
+  border: 0;
+  padding: 0;
+  margin: 0;
   display: flex;
   justify-content: center;
   align-items: center;
 
-  &:hover {
-    color: inherit;
+  &:focus {
+    outline: none;
+    & ${Icon} {
+      /* todo: how to show default outline on inner element cross-browser? */
+      outline: -webkit-focus-ring-color auto 5px;
+    }
   }
 
   ${({ isActive }) =>
     !isActive &&
     `
-  opacity: .4;
+  opacity: .5;
   &:hover { opacity: 1; }
   `}
 
   svg {
-    width: 22px;
-    height: 22px;
+    display: block;
+    width: 28px;
+    height: 28px;
   }
 `;
 
-function Media(props) {
-  return (
-    <Tab>
-      <Icon {...props}>
-        <MediaIcon />
-      </Icon>
-    </Tab>
-  );
-}
-
-function Text(props) {
-  return (
-    <Tab>
-      <Icon {...props}>
-        <TextIcon />
-      </Icon>
-    </Tab>
-  );
-}
+const Pane = styled.section.attrs(({ isActive }) => ({
+  role: 'tabpanel',
+  'aria-expanded': isActive,
+  hidden: !isActive,
+}))`
+  padding: 1em;
 
-function Shapes(props) {
-  return (
-    <Tab>
-      <Icon {...props}>
-        <ShapesIcon />
-      </Icon>
-    </Tab>
-  );
-}
-
-function Links(props) {
-  return (
-    <Tab>
-      <Icon {...props}>
-        <LinksIcon />
-      </Icon>
-    </Tab>
-  );
-}
+  ${({ hidden }) => hidden && `display: none;`}
```

Comment:
Why both `hidden` attribute and `display:none`?

Reply:
DIdn't actually know that browsers hid any element that was marked with `hidden`. But it does, so I removed it in f7756942. Thanks!